### DISCUSSION TITLE
Can't login from any device

### DISCUSSION THREAD

**Discussion by SergioRius** [nextcloud/all-in-one#5226] [Questions] [answered]
I've just ended migrating to AIO containers from a previous linuxserver docker install. I migrated the users and setup the same passwords they had. The install is apparently working right as I can login and explore the contents on the web interface.

If I try to login again (or remove account and try to re-login) from any mobile device I get an error about the credentials being invalid.
I generated an app credential for every device. I can reach the login page, then select alternative app login, then enter the user name and the generated guid, but always is incorrect.
I generated an app password for my computer and if I try to login, it says it's successful, but never stores the account and if I try to open the  interface it asks me to login again.

At some time during the install it seemed to me that something was wrong with the password validation. As I said I configured the same passwords as the old system, and that's the one I have stored in my bitwarden. But using the same bitwrden credentials failed. Even when I checked that the password matched.

I think several things are going on at the same time. The instance seems to not getting the device IP, but the proxy one. So it's systematically blocking the firewall/proxy IP, thus negating login to everyone. I currently have all the devices in airplane mode, and launching `security:bruteforce:reset` commands repeatedly.
I've tried adding the `'forwarded_for_headers' => array('HTTP_X_FORWARDED_FOR'),` to nextcloud container configuration, (config.php) but it has no effect.

Please help.

EDIT: 
I've already setup forwardfor headers in my proxy.
I've looked at the DB for entries at bruteforce table but it's empty. I don't know what's going on, then.

**Comment by SergioRius** [comment]
Now managed to connect one users smartphone apps to the server, by first login the nextcloud client using the QR code.
I've observed that trying to validate a password always fails.
The notes app stills throws an `android.accounts.NetworkErrorException` when trying to log on.
And if a login screen is the only method (like with memories) I always get a "Denied access. CSRF check failed."

**Comment by szaimen** [comment, marked as answer]
Hi, please opena thread for this at https://help.nextcloud.com
